### DISCUSSION TITLE
Idea on how to make EZ easier

### DISCUSSION THREAD

**Discussion by OliBomby** [ppy/osu#15376] [Ideas]
I saw this mentioned in the recent Community Meeting and I got an idea so I figured I'd suggest it here. 

I believe the reason why the Easy mod is actually harder is solely because of the reduced approach rate which increases the object density and makes the map hard to read. I also assume that the initial intention of the reduced approach rate in the Easy mod was to make the approach rate easier/more comfortable. Therefore I suggest using the mod customisation menu to let the user customize the approach rate within a range of 0.5x to 1.5x the original approach rate, similar to how the Difficulty Adjust mod works. This is guaranteed to always make the approach rate easier as the user can choose the approach rate which is most comfortable to them.

Also, considering this will significantly change the Easy mod, it might be worth making the original Easy mod with the forced reduced approach rate a separate mod so people can still compete for the high object density reading type of skill.

**Comment by Moltanicaa** [comment]
You can also make it so that AR above 9 will not decrease at all, and AR below 9 will decrease in the standard way.

**Reply by OliBomby** [reply]
> You can also make it so that AR above 9 will not decrease at all, and AR below 9 will decrease in the standard way.

I dont like this approach because beatmaps with AR below 9 are not necessarily easier with even lower AR.

**Reply by Ezius** [reply]
The biggest problems is the AR because it just gets too crowded ,so I would say the approach rate should be based on the average amount of jump density the map has for example if there is a 1/2 jump section the average should be 3 to 4 objects (I think?) on screen but that would probably be really hard to implement, because of 1/4 and 1/3rd mapping or bpm changes in a song and some other flaws which could probably be ironed out over time though.

I feel like OliBombys suggestion is a bit redundant because its basically just Easy mod + Difficulty adjusted AR

But my solution is also not perfect so I hope we can gather more ideas here

**Reply by OliBomby** [reply]
> The biggest problems is the AR because it just gets too crowded ,so I would say the approach rate should be based on the average amount of jump density the map has for example if there is a 1/2 jump section the average should be 3 to 4 objects (I think?) on screen but that would probably be really hard to implement, because of 1/4 and 1/3rd mapping or bpm changes in a song and some other flaws which could probably be ironed out over time though.

It's basically impossible to make an algorithm that finds the easiest AR for a beatmap, because this also depends on the player. Some players might read low AR most comfortably while other players might read high AR most comfortably.

> I feel like OliBombys suggestion is a bit redundant because its basically just Easy mod + Difficulty adjusted AR

I don't think its redundant, because I think using Easy + Difficulty Adjust will have more impact on your score multiplier than just using Easy mod. It's only redundant if the AR Difficulty Adjust mod has an exception where it doesn't decrease score multiplier when used in conjunction with the Easy mod.
